### ISSUE TITLE
error messages for failed logins and failed port updates don´t work

### ISSUE THREAD

**Issue by jagaimoworks** [patrickaclark/gluetun-qbittorrent-port-manager#6]
The return codes of the curl statements in the update_port function do not reflect the errors they are trying to catch. If the qBittorrent  API responds  successfully the curl return code will always be 0. To check for failed logins and port updates one would have to look at the API's responses instead.

**Comment by patrickaclark** [comment]
Makes sense. , I mainly made it as a quick check for connectivity and initial setup.  Didn't really use the right method except being a quick and dirty connection check, probably could have just used a ping.  Might look at this later on, but I have a feeling this will become redundant, as I believe gluetun might actually be adding some functionality for this down the road, making this package redundant.  But if someone wants to throw in some API check code, I would not be against that.

**Comment by RelicCornhusk** [comment]
I've created a PR for that: #7
